apps/demo-game/src/pages/play/cockpit.tsx

```
@@ -289,7 +287,7 @@ function Cockpit() {
         }
       })
 
-      const labels = ['Bank', 'Bonds', 'Stocks']
+      const labels = ['Savings', 'Bonds', 'Stocks']
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)